odoo/addons/base/views/res_partner_views.xml

```diff
@@ -339,9 +339,14 @@
                                         <field name="parent_id" invisible="1"/>
                                         <hr/>
                                         <group>
-                                            <group attrs="{'invisible': [('type','=', 'contact')]}">
-                                                <label for="street" string="Address"/>
-                                                <div>
+                                            <group style="width:41%">
```

Comment:
width 41%? seems like a bad idea, use col and colspan

Reply:
Hello @mart-e , 
If we use col and colspan instead of the static width, information gets cut as information is not fully displayed. that's the reason behind using it.
cc @mba-odoo

Reply:
@jso-odoo I think I have found another way with col and colspan, can you check my last commit?